packages/cursorless-vscode-core/StatusBarItem.ts

```diff
@@ -0,0 +1,35 @@
+import * as vscode from "vscode";
```

Comment:
simplified the status bar class by moving the command definitions into another file; they didn't really belong here